Source/SysVita/main.cpp

```diff
@@ -243,6 +304,8 @@ int main(int argc, char* argv[])
 		sprintf(fullpath, "%s%s", DAEDALUS_VITA_PATH("Roms/"), rom);
 		EnableMenuButtons(false);
 		System_Open(fullpath);
+		loadConfig();
```

Comment:
Ideally we need a glocal config too that will be used if a per-game config is missing. When closing a rom, it should also switch back to global config.